app/controllers/cloud_volume_controller.rb

```diff
@@ -203,7 +203,7 @@ def create
       @volume = CloudVolume.new
       options = form_params_create
       ext_management_system = options.delete(:ems)
-      if ext_management_system.supports?(:cloud_volume_create)
+      if ext_management_system.supports?(:create)
```

Comment:
I don't think this is the same thing?  Think you want `ext_management_system.class_by_ems(CloudVolume)`.supports?(:create)`